### ISSUE TITLE
Uninstall dynamo 2.13 for Revit 2023

### ISSUE THREAD

**Issue by RickBIM** [bvn-architecture/RevitBatchProcessor#112]
I don't know if here is the right place to make a question, but i'm trying to use the RBP on Revit 2023, but with the updates Revit install two versions of Dynamo 2.13 and 2.16. How do I uninstall the 2.13 version?

**Comment by JinminLi0430** [comment]
Hi Rick, I think I got the same issue with you that the Batch process can't process the dynamo script but can run the Python script. I think it could be the dynamo version issue. Wondering you have you found the solution for it to uninstall the dynamo version?

Thanks

**Comment by RickBIM** [comment]
Yeah... Don't find a solution, but the Dynamo Multiplayer of BirdTools worked for my needs: https://www.birdtools-developers.com/dmu.html

**Comment by JinminLi0430** [comment]
> Yeah... Don't find a solution, but the Dynamo Multiplayer of BirdTools worked for my needs: https://www.birdtools-developers.com/dmu.html

Thank you... It is super helpful, will check it out. If it can be run outside of Revit such as this Batch Revit Browser would be perfect.
